pypeman/contrib/hl7.py

```diff
@@ -72,6 +76,7 @@ def connection_lost(self, exc):
         meaning a regular EOF is received or the connection was
         aborted or closed).
         """
+        logger.info("MLLP: connection lost (exc=%r)", exc)
```

Comment:
to avoid pickle issues with socket loggers:
`logger.info("MLLP: connection lost (exc=%s)", repr(exc))`

Reply:
done